### ISSUE TITLE
Error while using Attendee mover with decaf

### ISSUE THREAD

**Issue by Saam01** [eventespresso/event-espresso-core#3807]
When I am using Attendee mover with decaf, it is showing the below error:

![image](https://user-images.githubusercontent.com/99984614/163811230-b2060d02-c6d9-4a8b-a3e0-3e35a4002048.png)
![Error](https://user-images.githubusercontent.com/99984614/163811461-ee24ffe8-85aa-4817-b549-954758b233b0.png)

**Comment by garthkoyle** [comment]
Yes, I can confirm the same fatal. Is this intentional @Pebblo or @tn3rb ?
![image](https://user-images.githubusercontent.com/3094547/163889649-25d4251f-c802-4983-ac61-e82892c64304.png)

**Comment by garthkoyle** [comment]
@Pebblo and @tn3rb is the attendee mover supposed to work with Decaf or not? If not, should it completely fatal like that?

**Comment by tn3rb** [comment]
i don't think it should work with decaf but it should also not fatal like that.
so definitely a bug in the attendee mover

**Comment by garthkoyle** [comment]
Ya, agreed.

yes, that’s what I’m hoping. But I don’t know if that is the case with the Attendee Mover specifically. Some add-ons do work with Decaf, and I want to be sure Attendee Mover does not. But I’m uncertain because it just throws a Fatal Error rather than something more intelligent saying that a different version of EE is required.

**Comment by Pebblo** [comment]
So afaik, with regards to Decaf we have always considered add-ons to not work with it, some may do, but for as long as I can remember we've always advised that to use the add-ons you need to use the full version of EE4.

Now, having said that, the above I would still consider a bug as it's adding the icon then just throwing a fatal when you use it, the add-on should just not add its functionality at all if that is possible here.

**Comment by garthkoyle** [comment]
Thanks. It's a bug with Decaf then.
